src/routing/router.ts

```diff
@@ -18,17 +16,13 @@ export class Router<
 
   constructor(protected _routes: R[]) {}
 
-  protected static getRouteMetaData(
-    route: Route<Middleware, Controller>
-  ): { [key: string]: any } {
+  protected static getRouteMetaData(route: Route<Middleware, Controller>): { [key: string]: any } {
     /**
      * controller and middleware are constructors
      * there should be no need for them
      */
     const isUsefulKey = (key: string) =>
-      typeof (route as any)[key] !== 'undefined' &&
-      key !== 'controller' &&
-      key !== 'middleware'
+      typeof (route as any)[key] !== 'undefined' && key !== 'controller' && key !== 'middleware'
```

Comment:
check out 7.1.2 here http://exploringjs.com/es6/ch_symbols.html

Reply:
not a bad idea but outside the scope of this change

Reply:
feel free to open an issue or pr or both 😉